src/migrations/20201016082749-create-line-managers.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
i guess line manager could be user m should we need this table or this can be a field of another table? lets brainstorm on this

Reply:
I was wondering how we will have users and managers tables